documentation/faq/70-packages.md

```diff
@@ -2,8 +2,8 @@
 question: How do I fix the error I'm getting trying to include a package?
 ---
 
-Old beta versions of the SvelteKit template included the configuration value `noExternal: Object.keys(pkg.dependencies || {})` in `svelte.config.cjs`. First, please check if this line is present in your project and remove it if so. Removing this line fixes most issues and has since been removed from the template.
+Old beta versions of the SvelteKit template included the configuration value `noExternal: Object.keys(pkg.dependencies || {})` in `svelte.config.js`. First, please check if this line is present in your project and remove it if so. Removing this line fixes most issues and has since been removed from the template.
```

Comment:
I would add a sentence here like "if you really need to import the package.json, you first need to create a require to import JSON files as ESM does not support these yet without a flag: \<code snippet\>"

Reply:
good idea, done